anda/devs/tmon/tmon.spec

```diff
@@ -0,0 +1,46 @@
+%global commit e8cae0d88dc8d51cb0e34e3fd3553bcbdaf04ca5
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+%global commit_date 20241215
+%global debug_package %{nil}
+
+Name:           tmon
+Version:        %{commit_date}.git~%{shortcommit}
+Release:        1%{?dist}
+Summary:        A tiny system monitor for Linux
+
+License:        GPL-3.0
+URL:            https://github.com/pondda/tmon
+Source0:        %{url}/archive/%{commit}.tar.gz
+
+Requires:       lm_sensors
+Conflicts:      kernel-tools
+
+Suggests:       nerd-fonts
```

Comment:
`Suggests` is useless here I think

Reply:
Can't really tell if it needs an emoji font or nerd font installed tbh

Reply:
It will work without either because it can be used in a tty, but I would prefer to suggest/recommend one of the two, or perhaps even both? What do you think?

Reply:
`Suggests:` will have no effect on this
`Recommends:` will pull in all `nerd-fonts`
soooooo *dunno*

Reply:
ah i see what you mean now